Source/player.cpp

```diff
@@ -625,7 +625,7 @@ void CreatePlayer(int pnum, char c)
 		plr[pnum]._pgfxnum = ANIM_ID_STAFF;
 	}
 
-	for (i = 0; i < sizeof(plr[pnum]._pLvlVisited); i++) {
+	for (i = 0; i < NUMLEVELS; i++) {

```

Comment:
@galaxyhaxz Do you remember if `_pLvlVisited` was used in beta or some such? I remember one version having `10` and another `17` for this value.

Reply:
That was the case in the PSX symbols. A few of them used 10 and others 17. All versions of Diablo used 17, and Hellfire used 25. (it was `_pSLvlVisited` that only had 10 used)

Reply:
> That was the case in the PSX symbols. A few of them used 10 and others 17. All versions of Diablo on PC used 17, and Hellfire used 25. (it was `_pSLvlVisited` that only had 10 used)

So, should we use `NUMLEVELS` or `sizeof(plr[pnum]._pLvlVisited) / sizeof(plr[pnum]._pLvlVisited[0])` for this one?

Reply:
I guess `NUMLEVELS` is fine.